src/components/DisplayNames/DisplayNamesWithTooltip.tsx

```diff
@@ -46,13 +46,14 @@ function DisplayNamesWithToolTip(props) {
     return (
         // Tokenization of string only support prop numberOfLines on Web
         <Text
-            style={[...props.textStyles, styles.pRelative, props.numberOfLines === 1 ? styles.noWrap : {}]}
-            numberOfLines={props.numberOfLines || undefined}
-            ref={(el) => (containerRef.current = el)}
+            style={[textStyles, styles.pRelative, numberOfLines === 1 ? styles.noWrap : {}]}
```

Comment:
Just wondering why the spread syntax was removed here?

Reply:
@jjcoffee This way `textStyles` folllows `StyleProp<TextStyle>` type and user can provide an array or object, and both will work cause React Native supports nested arrays in styles.